src/modules/task.test.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @jest-environment jsdom
+ */
+const updateTaskStatus = require('./task.js');
+
+describe('Should if the status is toggling: ', () => {
+  test('should update completed status', () => {
+    expect(() => {
+      updateTaskStatus('<input type="checkbox" name="checkbox" class="checkbox" id="Read bookssss">', {
+        index: 1,
+        description: 'Read book',
+        completed: true,
+      });
+    }).toBeTruthy();
```

Comment:
- [ ] As the previous reviewer asked, kindly check that the status of the task has changed to the new value. Your function returns `true` at the end so your test will be truthy no matter what... Kindly rewrite the test to match what we want to test which is the status of the task 👍